src/backend/renderer/damage.rs

```diff
@@ -254,17 +262,17 @@ impl TryInto<(Size<i32, Physical>, Scale<f64>, Transform)> for DamageTrackedRend
     }
 }
 
-/// Damage tracked renderer for a single output
+/// Damage tracker for a single output
 #[derive(Debug)]
-pub struct DamageTrackedRenderer {
-    mode: DamageTrackedRendererMode,
+pub struct OutputDamageTracker {
+    mode: OutputDamageTrackerMode,
     last_state: RendererState,
     span: tracing::Span,
 }
 
-/// Errors thrown by [`DamageTrackedRenderer::render_output`]
+/// Errors thrown by [`OutputDamageTracker::render_output`]
 #[derive(thiserror::Error)]
-pub enum DamageTrackedRendererError<R: Renderer> {
+pub enum OutputDamageTrackerError<R: Renderer> {
```

Comment:
Actually, while we are at it (sorry for bringing this up only just now)... I would vastly prefer to just name this `Error` and solve conflicts by renaming the import in other modules.

This is not C, we don't have to prefix everything with the module-name or similar.

Reply:
I'd agree with this, but then I ask that question about the rest of smithay. Should Gles2Renderer be gles2::Renderer or do we want the explicit name. Some of the reexports add a prefix (gbm::Device exported as GbmDevice)

Reply:
I am fine with changing it, but we should stay consistent. So if we change it here we should also it everywhere else. Not sure if gradually, starting with this one, or changing all at once is better. But I lean towards making that change gradually. This could potentially affect a big surface of the API. No one likes to review such big PRs and it might take a bit so the potential for rebase conflicts is pretty high.

Reply:
Right and at least for error types I think we are semi-consistent in the sense, that we prefer shorter names.
Also I don't think for main structs like the `Gles2Renderer` or the `OutputDamageTracker` renaming really makes sense. For the Gles2Renderer it is particularly problematic given the `Renderer`-trait just one module above.

Reply:
renamed to `Error`